pkg/asset/manifests/capiutils/helpers.go

```diff
@@ -19,11 +18,6 @@ func CIDRFromInstallConfig(installConfig *installconfig.InstallConfig) *ipnet.IP
 	return defaultCIDR
 }
 
-// IsEnabled returns true if the feature gate is enabled.
-func IsEnabled(installConfig *installconfig.InstallConfig) bool {
-	return installConfig.Config.EnabledFeatureGates().Enabled(v1.FeatureGateClusterAPIInstall)
-}
-
```

Comment:
Instead of deleting this, why not: 
```go
// IsEnabled returns true if the feature gate is enabled.
func IsEnabled(installConfig *installconfig.InstallConfig) bool {
	platform := installConfig.Config.Platform.Name()
	if azure := installConfig.Config.Platform.Azure; azure != nil && azure.CloudName == azuretypes.StackCloud {
		platform = azuretypes.StackTerraformName
	}
	return types.ClusterAPIFeatureGateEnabled(installConfig.Config.Platform.Name(), installConfig.Config.EnabledFeatureGates())
}
```
instead of duplicating this logic around.

Reply:
much better. incorporated.